data/sports/football/world_cup_history.json

```diff
@@ -0,0 +1,173 @@
+{
+ "description": "FIFA World Cup stats organized by year. Includes organizer, winner, runners-up, third and fourth. Data from FIFA scores, Sept. 2018.",
```

Comment:
Is there a source where you compiled this from? You can add it as an additiona property to the root JSON with:

```javascript
  "source": "url",
```

Reply:
great idea!
solved!